regform.html

```diff
@@ -0,0 +1,172 @@
+<!doctype html>
+<html>
+<head>
+    <title> Registration Form</title>
+    <style>
+         #basicuserinfo {
```

Comment:
Why repeating same styles multiple times? 
You can use complex selectors: 
`#basicuserinfo, #addressinfo, ... {styles}`